kubernetes/cray-console-node/templates/hook-postupgrade.yaml

```diff
@@ -38,7 +38,7 @@ spec:
       - name: hook1-container
         image: {{ .Values.alpine.image.repository }}:{{ .Values.alpine.image.tag }}
         imagePullPolicy: {{ .Values.alpine.image.pullPolicy }}
-        command: ['sh', '-c', 'mkdir -p /var/log/conman /var/log/conman.old /var/log/console && chown -Rv 65534:65534 /var/log && chmod -R 700 /var/log']
+        command: ['sh', '-c', 'mkdir -p /var/log/conman /var/log/conman.old /var/log/console && chown -Rv 65534:65534 /var/log && chmod -R 777 /var/log']
```

Comment:
Why are we opening the permissions up wider here? 700-->777? This seems consistent with the 777 permissions below. I was just wondering why it had to change from 700 to 777.

Reply:
Didn't HAVE to change, I updated it to match the permissions of the files. Just consistency.